.travis.yml

```diff
@@ -1,20 +1,26 @@
-jdk:
-- oraclejdk8
-before_install:
-- sudo apt-get install libaio1
-script: mvn verify
-after_success:
-- test "${TRAVIS_PULL_REQUEST}" == "false" && test "${TRAVIS_TAG}" != "" && mvn deploy
-  --settings travis/settings.xml
-cache:
-  directories:
-  - "~/.m2"
-branches:
-  only:
-  - master
-  - "/^elide-parent-pom-[0-9]+\\.[0-9]+\\.[0-9]+\\.[0-9]+/"
-env:
-  global:
-  - secure: VV+uJlBgXRCUSj3+khXnKvbrB0bxtay3yovKAchTT83Y6+iXUAgp3b7EA1QjT0l3Hz88Y+EAVuiZqwzFRidbZwbgApaxYMJKWjHTmr+VsXXJp29J1tyGtXR1K+DUG0HO/jiRCxMVythB68BnNMIXfprpmsLFpXhKMPf41fzMaAtqSYrPdJkNvEzBDhu5hIPG9apyEcpIZNhtfwnk18DntyGFC3OVAKzEma51s0tS6NeXiZNzTvc9j0d4Fr8vyWGclAp1XKMjWzpvfRyB2EFto076V+8tJ5E3czJjJs12SDwOvEdzYeCpGk3VWboiz9DuZn41IdTLYy89OmcIooZQHViMwiatcqqHREEbuuUWZLYMEh1bUHZHRWQf2AnUScDfJB1QUKVR8mbkSr5DXto4FIsHpqofoBloDveaXkWC735DqOPwbIwe2owGZfkQv19vPpVvF3e2LyFn1vf8blR3wFBItKKZyvBTbrIVM/YxuTDfswDVlhEgHTeet0TQSy4yEoMxGTHsfk8Ykc2B0JDheWH5zhhZBdozmV5TFbM50PXjLqfqWVDD+WqzGVer1Sokfv6MMERoWKvb/LEyKO4Eq3aHgXyy2GoRQHfUh9qOOvBnNpHbtacL+NzfggLZ1Ut97QIbEoWb3VPx6lbNKxiiC4DbwR7nnc92zNJM5YU6Dog=
-  - secure: SSqm3fI2slK6qcmfeRiWvQJ8kNd5VQD6wSkFM5NMCo7Uvo3peNNwCDiTpWMN6nNuM4LSJJhgGwDLZeIlRF4i74xgWpDznjZ4AoqgG1QeIjg/4Xdy+szz0+BiTdUhQnRsaKHfUoky4wm+Jrix1rYgz/eED6uiOHmgLWywD4+BEBHN37N5N6/vT7ELkxi/jzXWOunMA12Y6F+LFcw4A87qM94iFws94C0BEqfWRqaXYPZoQ7v5Yw7TilJuPGiL6pcdWyh5ObfAhIbHhhxmIvaMFWV7HGoB1rC2T/E/GQynKU/l1uQHTf/rNK70uUTCKz3usl28AnVsnyhjg7+Ivk22Z0pSnZtLw1TJT5prPdX+Cas8NEElk6zSS/h3lMmQuGZk8T6FmsqkDx3tkFiJqgxvbgGLZBMgYdm87OCzrM55ZTlxBOYErQQgFrevJ2oJV98Pb4DNCPx5uu/CVCbR+d1kZ6b91rabzjl+a3yiz10vnaO00dyW1gR1P0KHZiRactdds2jLsxBhr3MFDymZD7W+WhuI4RtP2/bsQQ1dnLIvWm+JYmDvr09/7k1uX7WCm9AkMb0u1gW19WYQT1Qct3hFfvKIbgv3gTNJz2+Alu3mqeIpSkmtdtGjVMij3/0OBdYBNC+BfaR4IQVnIbvyu0HmjtJgRji2TO/U7rCUKjOrm6c=
+---
+  language: java
+  jdk:
+    - oraclejdk8
+  cache:
+    directories:
+      - "~/.m2"
+  branches:
+    only:
+      - master
+
+  # build steps
+  before_install:
+    - sudo apt-get install libaio1
+  script: mvn verify
+  before_deploy: cat travis/bintray_template.json | sed s/\$\{TRAVIS_TAG\}/${TRAVIS_TAG}/ > travis/release.json
+  deploy:
+    on:
+      tags: true
+    provider: bintray
+    file: travis/release.json
+    user: clayreimann
+    key:
+      secure: eszSq9UgLy9bd2j6Z+zQvXAUnnEZy+DBcyjgzE26EA2cSZhAU+QdSkHgFsbU10fb9nRUK0mbunH/BxP/nMEtoVWRa33ALUfljvLMvdO+gOpqIWsdtmsvIn+76t774zNz90NMTtZuDWknbwKz1zP3ysAmj7KFtJIJAXpuDtvGyVmNyZ6z/qFJQCuHhSEsQf70cS+EZXLEkH16f+tSFQwwW4x4ucQY0qHbeiCVw7meCi8zue2/CwFcNw9wVukXOQ+zYScQStHPksce9+9OH9DlaeWB0EgKm7LWuTfip2O8ebpoyv5mOaAag5w72I82xP76GWxHFlFVr/RSs/Uudqgugc5lsy2Myh+xDKQgurdLfn6+7lF7F23ola7q9NgVnyG7vRxYUhPp6DL/a/gFqAfD2nDhz/kcZQnGEPh+Ookt1KJv3PaI2gpDeACAvXH04vG4Y5ZQaP/wC7hzADRYCsJPaHWumA7vVzmB4VTxsAuoAJaOs0iu99RrS+wKW/kGKQFh3wUxuVQ3W+EfySFjBzGVsTEcqH2ZX4xwFDR/nHOWI35KpGlcyEVVr+M52Of6raacn9nAv2EQxGuRIqL8yI9zMzicMVT1syl64slh3BSIVLZfU75j/CmSFHL7CKRE+A14CiXFkGdtg23+UQPGVU4LnaSl0/JnO+7OlDiqX1qxbyM=
+    dry-run: true
+  after_deploy: echo "Pending"
```

Comment:
It is indeed. But I want to verify that bintray is getting the right artifacts first, which means a new release.